src/main/java/com/knu/buga1chuk/algo/search/BinarySearch.java

```diff
@@ -1,14 +1,21 @@
 package com.knu.buga1chuk.algo.search;
 
-public class BinerySearch {
+import com.knu.buga1chuk.serialization.json.ReadPersonJsonDemo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BinarySearch {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ReadPersonJsonDemo.class);
+
     /**
      * PrintSearchResult
      */
     public static void printSearchResult(int numberToFind, int idx) {
         String messagePattern = "Element %d is located on index %d";
         String result = String.format(messagePattern, numberToFind, idx);
-        System.out.println(result);
-
+        //System.out.println(result);
+        LOG.info(result);
```

Comment:
Simplify `messagePattern` creation with the string interpolation. 
(It is one of the key and useful feature of the logger)

Refactor it here and in other places.

Reply:
fix